projects/ekg.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name_cz": "Prevence arytmií",
-  "name_en": "Arrhythmia prevention",
+  "name_cz": "Prevence srdečních arytmií",
+  "name_en": "Heart arrhythmia prevention",
   "subtitle_cz": "Brzká dektekce arytmie z EKG",
   "subtitle_en": "Early arrhythmia detection from ECG",
+  "website": "mild.blue",
```

Comment:
Asi bych sem nepsal žádný web. Myslel jsem to tak že když tu web bude tak na stránkách budeme mít details here. Takhle není poznat který projekt má svou webovou a co kterýne..

Reply:
hele, klidne to smaz, je to fakt jedno